package.json

```diff
@@ -46,7 +46,6 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "commander": "^5.1.0",
     "debug": "^4.1.1",
```

Comment:
there's also a "bin" field that should be removed.

Reply:
Done!